tutorials/remove all entity shadows

```diff
@@ -0,0 +1,327 @@
+**Add / Remove all shadows by code**
+
+by Outlandishly Crafted updated by Jeremy Benisek AKA CyberAxe
+06212020
+
+#How to remove all entity shadows:
+
+Change these lines in the shadow.material file under the Resource Pack / materials folder. 
+NOTE: This folder is NOT included in the vanilla RP Pack examples and must be exported from a APK files or added by hand.
+NOTE: This has not been tested for blocks and has only been verified for entities. If you find it works on blocks too please let us know so we can add that in.
+
+Working shadow code: Shadows for ALL entities:
+"shadow_overlay": {
+      "+states": [
+        "DisableDepthTest",
+        "DisableCulling",
+        "Blending",
+        "EnableStencilTest"
+      ],
+
+      "vertexShader": "shaders/color.vertex",
+      "vrGeometryShader": "shaders/color.geometry",
+      "fragmentShader": "shaders/shadow_stencil_overlay.fragment",
+
+      "blendSrc": "DestColor",
+      "blendDst": "Zero",
+
+      "frontFace": {
+        "stencilFunc": "Equal",
+        "stencilPass": "Replace"
+      },
+      
+Disabled shadow code: No Shadows for ALL entities: 
+ "shadow_overlay": {
+      "+states": [
+        "DisableDepthTest",
+        "DisableCulling",
+        "Blending",
+        "EnableStencilTest"
+      ],
+
+      "vertexShader": "",
+      "vrGeometryShader": "",
+      "fragmentShader": "",
+
+      "blendSrc": "DestColor",
+      "blendDst": "Zero",
+
+      "frontFace": {
+        "stencilFunc": "Equal",
+        "stencilPass": "Replace"
+      },
+
+Here is an example from vanilla of the shadow.material file with working shadows
+//Edited by Outlandishly Crafted by CyberAxe Laston:062020 Version:v001 Topic:Enable All Shadows
```

Comment:
Once again, credit should be in the footer. This is just to keep the wiki as clean as possible. Also keep in mind the commit itself will be shown as under your account, so you also get some recognition that way.